packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/create/template.html

```diff
@@ -0,0 +1,14 @@
+<div class="text-center" data-ng-if="$ctrl.loading">
+    <oui-spinner></oui-spinner>
+</div>
+
+<div data-ng-if="!$ctrl.loading">
+    <octavia-load-balancer-pool-form
```

Comment:
`data-ng-if` could be set directly on `octavia-load-balancer-pool-form`. `<div>` is not really ncessary

Reply:
Strange I have a class="text-center" on this div that does not appear here O_o